src/lib/mongoBackend/mongoRegistrationGet.cpp

```diff
@@ -205,9 +205,9 @@ static bool setDataProvided(ngsiv2::Registration* regP, const mongo::BSONObj& r,
 *
 * setExpires -
 */
-static void setExpires(ngsiv2::Registration* regP, const mongo::BSONObj& r)
+static void setExpires(ngsiv2::Registration* regP, const mongo::BSONObj& r, const char* dbLabel)
```

Comment:
Function is overparametriced, `dbLabel` is not needed at the field used for expired is not a parameter but fixed by the database model.

Reply:
Fixed in 5d851ef